docs/conf.py

```diff
@@ -57,3 +57,10 @@
 
 # This value contains a list of modules to be mocked up.
 autodoc_mock_imports = ["camel_converter"]
+
+html_title = 'Meilisearch Python | Documentation'
+
+# Add Fathom analytics script
+html_js_files = [
+    ("https://cdn.usefathom.com/script.js", { "data-site": "QNBPJXIV", "defer": "defer" })
```

Comment:
I don't know anything about fathom, I assume this is a key `"data-site": "QNBPJXIV"`? Is it OK to share publically?

Reply:
Hello @sanders41 👋 

It is OK to share this key, as it will end up in the HTML anyway.